lightning-network-book.tex

```diff
@@ -373,25 +373,24 @@ \section{Introduction to Electronic Cash}
 From a technical point of view they are just entries in a third party's database.
 Thus, they are based on trusting this third party.
 If your digital cash service provider decides to deny you access to your funds or run away with your money, there is basically nothing you can do.\footnote{Of course in most jurisdictions there is the legal system preventing them from stealing your money.}
-Such a scenario can easily happen as we have seen with the bankruptcy of the famous bank Lehman Brothers in the summer of 2008. Many people that trusted their cash to this bank lost a fortune.
-While we have to be fair and attest that it was certainly not the fault of their digital and virtual cash systems that they filed for bankruptcy, we see that trusting a bank to have a digital form of cash removes the most important property of cash.
-The fact that you (and only you) will own it.
-But the reason banks existed was not only to be a provider of a digital form of cash.
-They already existed before the digital age because people did not want to carry around vast amounts of money.
+Such a scenario can easily happen as we have seen with the bankruptcy of the famous Lehman Brothers bank in the summer of 2008. Many people that trusted their cash to this bank lost a fortune.
+While we have to be fair and attest that it was certainly not the fault of their digital and virtual cash systems that they filed for bankruptcy.
+We see that trusting a bank to have a digital form of cash removes the most important property of cash: the fact that you (and only you) will own it.
+
+So the question remains: Can there be a better form of digital cash compared to the solution provided by the traditional banking industry? 
```

Comment:
not sure if I agree with this particular suggestion. the frame changes quite a bit. Will merge this however and see what others think on the long term.